src/main/java/org/benetech/servicenet/repository/LocationMatchRepository.java

```diff
@@ -16,7 +18,10 @@
 public interface LocationMatchRepository extends JpaRepository<LocationMatch, UUID> {
     Set<LocationMatch> findByLocationId(UUID locationId);
 
-    Optional<LocationMatch> findByLocationIdAndMatchingLocationId(UUID locationId, UUID matchingLocationId);
+    @Query("SELECT locationMatch FROM LocationMatch locationMatch "
```

Comment:
is this any different from the jpa-generated method?

Reply:
It's not, but tests don't see without this.

Reply:
It's not, but tests don't see without this.